pkg/cmd/cli/secrets/satoken.go

```diff
@@ -0,0 +1,116 @@
+package secrets
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/client/unversioned"
+
+	cmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+)
+
+const (
+	ExtractSATokenRecommendedName = "extract-sa-token"
+
+	extractSATokenShort = `Extract the token from mountable secrets assigned to a ServiceAccount.`
```

Comment:
Use "service account" not "ServiceAccount".  Humans are reading this, not robot overlords (yet)